tools/sidekick/config.json

```diff
@@ -0,0 +1,13 @@
+{
+    "project": "24life",
```

Comment:
@synox  do we want to adjust the github project settings so PR is hinted to get the latest changes from main?

Reply:
The config is described here: https://www.hlx.live/docs/sidekick#customizing-the-sidekick

Reply:
Sorry, I meant this file config.json is already present in the main branch.  If the branch is up-to-date with main we wouldn't have seen this file is what I meant :)

Reply:
Ah there was a merge conflict, that's why. Should now look better.